template/config/index.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: './',
```

Comment:
As I tested, the empty string would be better. So that you can deploy it to any sub directory. And the empty string is the default value of web pack's `output.publicPath`, see https://webpack.js.org/configuration/output/#output-publicpath